### DISCUSSION TITLE
Conditionals

### DISCUSSION THREAD

**Discussion by azc666** [spatie/mjml-php#23] [Q&A]
Sending an email with spatie/mjml-php works for me. I can't figure out how to include conditionals though.
I tried with and without the handlbars <mj-raw>{{@if (...)}}</mj-raw><mj-text>...</mj-text><mj-raw>{{@endif}}</mj-raw>
What is the correct way to code this?

**Comment by amsoell** [comment]
Came here asking the same question. Not sure if this is the intended way, but I found that you need to run the MJML through the Blade renderer first, and then through this package:

```php
use Illuminate\Support\Facades\Blade;
use Spatie\Mjml\Mjml;

$mjml_template = file_get_contents('email/template.mjml');
$processed_mjml = Blade::render($mjml_template, [
    'header' => 'Here is the header',
    'body'   => 'And here is the body',
]);
$html = Mjml::new()->toHtml($processed_mjml);
```
